jib-core/src/main/java/com/google/cloud/tools/jib/api/Jib.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.api;
+// TODO: Move to com.google.cloud.tools.jib
```

Comment:
why not now?

Reply:
There are some other classes in `com.google.cloud.tools.jib` that shouldn't be there after when the library API is moved there. Once those are moved to their respective locations (or deleted), these `api` package classes should be moved there. This will help us make sure we don't forget to clean up the `com.google.cloud.tools.jib` package.